packages/backend/src/apps/m365-excel/common/constants.ts

```diff
@@ -3,3 +3,8 @@
 export const APP_KEY = 'm365-excel'
 
 export const MS_GRAPH_OAUTH_BASE_URL = 'https://login.microsoftonline.com'
+
+// This format is {UUID-VERSION-4}
+// Reference: https://stackoverflow.com/questions/7905929/how-to-test-valid-uuid-guid
+export const TABLE_ID_WITH_BRACES_REGEX =
+  /^\{[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\}$/
```

Comment:
Can we rely on a widely used trusted package for this, instead of trusting a comment on StackOverflow?

For example, [uuidv4's isUuid function](https://www.npmjs.com/package/uuidv4?activeTab=readme#verifying-a-uuid) or [validator's isUUID function](https://www.npmjs.com/package/validator)

Reply:
i wanted to use zod to validate the uuid but i needed the curly braces to be inside too, ian told me might as well just use a regex?

Reply:
OK - approving as UUID validation is relatively straightforward so there's little chance the regex could be wrong. But just note that it's usually poor practice to handroll validation code as it can be quite complex (e.g. email validation).

Reply:
i think we can be more lax with this, because im cant find any documentation that the table id will always follow UUID format? 
Also, can we take this chance to validate file id?

Reply:
ok but wah tbh i really cannot find documentation by Microsoft for table id and file id, so this is like validation based on the data we see on our db tho... is it actually ok